reddit_decider/__init__.py

```diff
@@ -214,6 +214,62 @@ def get_variant(
     #     self, experiment_name: str, variant_name: str, **exposure_kwargs: Optional[Dict[str, Any]]
     # ) -> None:
 
+    def _get_dynamic_config_value(
+        self,
+        dynamic_config_name: str,
+        decider_func: Callable[[str, DeciderContext], any],
+    ) -> Optional[any]:
+        context_fields = self._decider_context.to_dict()
+        ctx = rust_decider.make_ctx(context_fields)
+        ctx_err = ctx.err()
+        if ctx_err is not None:
+            logger.warning(f"Encountered error in rust_decider.make_ctx(): {ctx_err}")
+
+        res = decider_func(dynamic_config_name, ctx)
+        if res is None:
+            return
+        error = res.err()
+        if error:
+            logger.warning(f"Encountered error in decider.get_value(): {error}")
```

Comment:
best to specify which dc was called (get_bool(), etc) instead of generic `get_value()`)